raddb/clients.conf

```diff
@@ -1,6 +1,6 @@
 #  -*- text -*-
 #
-#  :toc:
+#  :toc: left
```

Comment:
If it's possible, this should go into either a style sheet, or maybe a command-line option.

Reply:
`asciidoc-a toc=left FILE`

Maybe we just want to set this as the default for all of the files.

Reply:
it was a typo. :(